.github/CODEOWNERS

```diff
@@ -0,0 +1,39 @@
+###################################
```

Comment:
Make codeowners for all files in this repo @georgi-l95(me), @isavov, @Ivo-Yankov, @natanasow, @konstantinabl and @Ivo-Yankov .

Reply:
Modified.

Reply:
Thank you 🚀